test/test-network/systemd-networkd-tests.py

```diff
@@ -3101,6 +3101,8 @@ def _test_activation_policy(self, test):
                 check_output('ip link set dev test1 down')
             expect_up = initial_up if always else next_up
             next_up = not next_up
+            if always:
+                time.sleep(1)
```

Comment:
Just wondering if there is there a more reliable way to tell if the state propagates to networkd? Sometimes the hardcoded sleeps aren't enough on a slow CI.

Reply:
Unfortunately, I do not have any good idea... Potentially, the following should work: invoke `journalctl --follow` before `ip link set`, and monitor the log until it contains `"ActivationPolicy is \"always-on\", forcing link up."`. But that sounds quite overkill, and it is had for me as I am not good at writing Python...

I'd like to keep this as is until the hardcoded sleep causes CI failures many times.